src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -194,253 +332,349 @@ const CurationPage = (props: ICurationPageProps) => {
+                                        name="propagationLiquidLevel"
+                                        options={[]}
+                                      />
+                                      <RealtimeTextAreaInput
+                                        fieldKey={`mutations/${mutation.firebaseIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/description`}
+                                        inputClass={styles.textarea}
+                                        label="Description of Evidence"
+                                        labelIcon={
+                                          <GeneHistoryTooltip
+                                            historyData={parsedHistoryList}
+                                            location={`${CANCER_TYPE_THERAPY_INDENTIFIER}${getMutationName(mutation)}, ${cancerTypeName}, ${
+                                              treatment.name
+                                            }`}
+                                          />
+                                        }
+                                        name="evidenceDescription"
+                                      />
+                                      <div className="mb-2">
+                                        <AutoParseRefField summary={treatment.description} />
+                                      </div>
+                                    </Collapsible>
+                                  );
+                                })
+                              );
+                            }, [])}
+                          </Collapsible>
+                        );
+                      })}
+                  </Collapsible>
+                </Col>
+              </Row>
+            ))}
           </div>
         )}
         <VusTable hugoSymbol={hugoSymbol} />
+        <Modal isOpen={showFilterModal} toggle={handleToggleFilterModal}>
+          <ModalHeader>
+            <Container>
+              <Row>
+                <Col>Filters</Col>
+              </Row>
+            </Container>
+          </ModalHeader>
+          <ModalBody>
+            <Container>
+              <h6 className="mb-2">Oncogenicity</h6>
+              <Row>
+                {tempOncogenicityFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        id={`oncogenicity-filter-${filter.label}`}
+                        onChange={() => handleFilterCheckboxChange(index, setTempOncogenicityFilter)}
+                        checked={filter.selected}
+                        disabled={!enabledCheckboxes.includes(filter.label)}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label for={`oncogenicity-filter-${filter.label}`} style={{ marginLeft: '18.75px' }}>
```

Comment:
Use class name instead of style?

Reply:
I wanted an exact left margin of 18.75 pixels. The checkbox originally comes with a left margin of -18.75px which I needed to set to 0 to fix the spacing. I use the above to offset this. I tried using `ml-3` and `ml-4`, but they both looked off to me. Let me know what you think:
![Screenshot 2024-01-02 at 4 54 45 PM](https://github.com/oncokb/oncokb-transcript/assets/148106517/ba87b070-b45d-47f9-afbd-e98612067cc4)
![Screenshot 2024-01-02 at 4 54 57 PM](https://github.com/oncokb/oncokb-transcript/assets/148106517/e9d5eef4-4f65-4d1b-b1a2-9ce87cba1c28)

Reply:
Makes sense. Then could you make it a constant and add a comment?